src/components/SlideDeck/SlideDeck.js

```diff
@@ -19,34 +19,36 @@ class Slide extends React.Component {
     }
 }
 
-class InvalidChildComponentError extends TypeError {}
+class InvalidChildComponentError extends TypeError { }
 
 class SlideDeck extends React.Component {
     static contextType = UxContext;
 
-    constructor(props){
+    constructor(props) {
         super(props);
         this.state = {
             currentSlide: 0,
             stickied: {
                 current: null,
                 previous: [],
             },
-         };
-
-        this.props.children.forEach(child => {
-            if(child.type.name !== "Slide" && process.env.NODE_ENV !== "production"){
-                throw new InvalidChildComponentError(`All children of SlideDeck must be Slide components. Got "${child.type.name}" instead`);
-            }
-        })
+        };
+        // This is not a good way to sanity check slides.
+        // It also breaks when there are less than 2 slides as children.
+        // You might want to embed static HTML or non-slide components.
+        // this.props.children.forEach(child => {
+        //     if (child.type.name !== "Slide" && process.env.NODE_ENV !== "production") {
+        //         throw new InvalidChildComponentError(`All children of SlideDeck must be Slide components. Got "${child.type.name}" instead`);
+        //     }
+        // })
```

Comment:
It breaks when there are less than 2 slides because react doesn't put the elements into an array if there is only one child. This is also an edge case that we don't really care about for this feature, as most of the time SlideDecks will contain more than one slide.

Please don't comment out working code. If you don't think it's a good idea, provide a replacement solution.